koku/api/apps.py

```diff
@@ -36,6 +36,7 @@ def ready(self):
         try:
             self.startup_status()
             self.check_and_create_service_admin()
+            self.get_or_create_public_schema()
```

Comment:
I would like to find another way to do this (and the service_admin bits, too).

The problem with hooking into `ready()` is that it executes when you do *anything* with the app, including attempting to tab-complete `manage.py` commands on the CLI. (I'm regularly irritated by this because I get `status` output anytime I try to tab-complete `manage.py makemigrations`  :unamused: )

Anyhow, if this *needs* to be here, fine. But if it can live anywhere else, I'd really like to see it not get implemented here.

Reply:
I will look into using a migration (specifically a "data migration" to add our schema tenant, and, if I can, the service admin.  https://docs.djangoproject.com/en/1.10/topics/migrations/#data-migrations